PowerEditor/visual.net/notepadPlus.sln

```diff
@@ -44,6 +76,28 @@ Global
 		{FBE04237-9C7B-4973-9C60-505975998B39}.Unicode Release|Win32.Build.0 = Unicode Release|Win32
 		{FBE04237-9C7B-4973-9C60-505975998B39}.Unicode Release|x64.ActiveCfg = Unicode Release|x64
 		{FBE04237-9C7B-4973-9C60-505975998B39}.Unicode Release|x64.Build.0 = Unicode Release|x64
+		{2581D488-956B-4D3B-BEE4-A9F1164759E2}.Debug|ARM64.ActiveCfg = Debug|Win32
+		{2581D488-956B-4D3B-BEE4-A9F1164759E2}.Debug|Win32.ActiveCfg = Debug|Win32
+		{2581D488-956B-4D3B-BEE4-A9F1164759E2}.Debug|Win32.Build.0 = Debug|Win32
+		{2581D488-956B-4D3B-BEE4-A9F1164759E2}.Debug|x64.ActiveCfg = Debug|x64
+		{2581D488-956B-4D3B-BEE4-A9F1164759E2}.Debug|x64.Build.0 = Debug|x64
+		{2581D488-956B-4D3B-BEE4-A9F1164759E2}.Release|ARM64.ActiveCfg = Release|Win32
+		{2581D488-956B-4D3B-BEE4-A9F1164759E2}.Release|Win32.ActiveCfg = Release|Win32
+		{2581D488-956B-4D3B-BEE4-A9F1164759E2}.Release|Win32.Build.0 = Release|Win32
+		{2581D488-956B-4D3B-BEE4-A9F1164759E2}.Release|x64.ActiveCfg = Release|x64
+		{2581D488-956B-4D3B-BEE4-A9F1164759E2}.Release|x64.Build.0 = Release|x64
+		{2581D488-956B-4D3B-BEE4-A9F1164759E2}.Unicode Debug|ARM64.ActiveCfg = Debug|Win32
+		{2581D488-956B-4D3B-BEE4-A9F1164759E2}.Unicode Debug|ARM64.Build.0 = Debug|Win32
+		{2581D488-956B-4D3B-BEE4-A9F1164759E2}.Unicode Debug|Win32.ActiveCfg = Debug|Win32
+		{2581D488-956B-4D3B-BEE4-A9F1164759E2}.Unicode Debug|Win32.Build.0 = Debug|Win32
+		{2581D488-956B-4D3B-BEE4-A9F1164759E2}.Unicode Debug|x64.ActiveCfg = Debug|x64
+		{2581D488-956B-4D3B-BEE4-A9F1164759E2}.Unicode Debug|x64.Build.0 = Debug|x64
+		{2581D488-956B-4D3B-BEE4-A9F1164759E2}.Unicode Release|ARM64.ActiveCfg = Debug|Win32
+		{2581D488-956B-4D3B-BEE4-A9F1164759E2}.Unicode Release|ARM64.Build.0 = Debug|Win32
+		{2581D488-956B-4D3B-BEE4-A9F1164759E2}.Unicode Release|Win32.ActiveCfg = Release|Win32
+		{2581D488-956B-4D3B-BEE4-A9F1164759E2}.Unicode Release|Win32.Build.0 = Release|Win32
+		{2581D488-956B-4D3B-BEE4-A9F1164759E2}.Unicode Release|x64.ActiveCfg = Release|x64
+		{2581D488-956B-4D3B-BEE4-A9F1164759E2}.Unicode Release|x64.Build.0 = Release|x64
```

Comment:
Why these above added lines?

Reply:
These are configurations for UnitTests projects generated by Visual Studio. I can try to reduce it.